connect/runtime/src/test/java/org/apache/kafka/connect/util/RetryUtilTest.java

```diff
@@ -50,16 +52,15 @@ public void setUp() throws Exception {
     @Test
     public void testSuccess() throws Exception {
         Mockito.when(mockCallable.call()).thenReturn("success");
-        assertEquals("success", RetryUtil.retryUntilTimeout(mockCallable, testMsg, Duration.ofMillis(100), 1));
+        assertEquals("success", RetryUtil.retryUntilTimeout(mockCallable, testMsg, Duration.ofMillis(100), 1, mockTime));
         Mockito.verify(mockCallable, Mockito.times(1)).call();
     }
 
-    // timeout the test after 1000ms if unable to complete within a reasonable time frame
-    @Test(timeout = 1000)
+    @Test
     public void testExhaustingRetries() throws Exception {
         Mockito.when(mockCallable.call()).thenThrow(new TimeoutException());
         ConnectException e = assertThrows(ConnectException.class,
```

Comment:
Nit: while we're in the neighborhood, can we remove this unnecessary variable? Can just change the line to:
```java
        assertThrows(ConnectException.class,
```

Reply:
I copied the error message assertion from a similar test, it looks like the original PR omitted it.